mkdocs/docs/index.md

```diff
@@ -1,22 +1,156 @@
 # IREE
 
-** * UNDER CONSTRUCTION ** *
+IREE (**I**ntermediate **R**epresentation **E**xecution **E**nvironment[^1]) is
+an [MLIR](https://mlir.llvm.org/)-based end-to-end compiler that lowers Machine
+Learning (ML) models to a unified IR optimized for real-time inference on
+mobile/edge devices against heterogeneous hardware accelerators. IREE also
+provides flexible deployment solutions for its compiled ML models.
 
-## Overview
+## Introduction
 
-<!-- TODO(??): copy introduction from README.md -->
-<!-- TODO(??): describe this website? -->
+### Key features
 
-## Components
+- [x] Support for dynamic shapes, flow control, streaming, and other advanced
+model features
+- [x] Ahead-of-time compilation of scheduling and execution logic together
+- [x] Low overhead, pipelined execution for efficient power and resource usage
+- [x] Binary size as low as 30KB on embedded systems
+- [x] Debugging and profiling support
+- [ ] Support for running quantized models (coming soon!)
```

Comment:
Not sure we need a separate point for quantized models. They are too detailed IMO and don't at the same level of points we are discussing here. (It is more fitting into the "Support for dynamic shapes, flow control, streaming, and other advanced model features" part.)

Reply:
SG. I'll remove that bullet. I thought it was worth calling out explicitly, since it's a special set of work that performance-focused users will look for on mobile.